dashboard/lib/pd/foorm/helper.rb

```diff
@@ -24,12 +24,35 @@ def get_survey_key(ws_submission)
     def get_index_for_survey_key(survey_key)
       if survey_key == "Pre Workshop"
         return 0
+      elsif survey_key.start_with? "Pre Workshop - Module"
+        # last character will be the module number, use that as an index
+        return survey_key[survey_key.length - 1].to_i
```

Comment:
Can you just use `survey_key[-1]` here?

Reply:
yep, I always forget you can do that in ruby